libdnf/rpm/solv_sack_impl.hpp

```diff
@@ -178,7 +178,7 @@ inline solv::SolvMap & SolvSack::Impl::get_solvables() {
     }
     // map.size is in bytes, << 3 multiplies the number with 8 and gives size in bits
     if (static_cast<int>(nsolvables) > (cached_solvables.map.size << 3)) {
-        cached_solvables = std::move(solv::SolvMap(static_cast<int>(nsolvables)));
+        cached_solvables = solv::SolvMap(static_cast<int>(nsolvables));
```

Comment:
Please could you explain why you want to remove std::move? there is move operator = , and it improves performance.

Reply:
```
../libdnf/rpm/solv_sack_impl.hpp:181:28: error: moving a temporary object prevents copy elision [-Werror,-Wpessimizing-move]
        cached_solvables = std::move(solv::SolvMap(static_cast<int>(nsolvables)));
```

Reply:
@j-mracek